picard/ui/mainwindow.py

```diff
@@ -902,6 +903,7 @@ def update_actions(self):
         self.cut_action.setEnabled(bool(self.selected_objects))
         files = self.get_selected_or_unmatched_files()
         self.tags_from_filenames_action.setEnabled(bool(files))
+        self.track_search_action.setEnabled(have_objects)
```

Comment:
I've changed the code to use `bool(sequence)`, but kept the intermediate variables because both `have_files` and `have_objects` are used twice.